### PR TITLE
traversal

### PR DESCRIPTION
**Pull request by mngmay**

### CODE REVIEW

**Comment by mngmay** [comment]
snitches get stitches

File: treasure-hunt/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "js-sha256": "^0.9.0",
```
Comment (AdnanCodes) [inline treasure-hunt/package.json:9]: Good work here finding this Package Kevin

File: treasure-hunt/src/App.js [hunk 2]
```diff
@@ -1,7 +1,68 @@
...
+import { mine, autoSnitchMiner } from "./actions/miningActions";
+
+import { axiosWithAuth } from "./util/axiosAuth";
 
 function App() {
-  return <div className="App"></div>;
+  // traverse(0);
+  // traverseDarkWorld(555);
+  // mine();
+  const [darkRoom, setDarkRoom] = useState(null);
```
Comment (AdnanCodes) [inline treasure-hunt/src/App.js:15]: Terrific work, setting up automated traversal

File: treasure-hunt/src/actions/miningActions.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+import { traverse } from "../util/shortestPath";
+
+export const getLastProof = () => {
+  return axiosWithAuth()
+    .get("bc/last_proof/")
+    .then(res => res.data)
+    .catch(err => console.log(err.response));
+};
+
+export const findProofOfWork = (lastProof, difficulty) => {
```
Comment (AdnanCodes) [inline treasure-hunt/src/actions/miningActions.js:16]: Kevin, awesome work figuring out the mining and setting up in a neat function

**Pull request merged**